### ISSUE TITLE
Attention robustness

### ISSUE THREAD

**Issue by alexdemartos** [Rayhane-mamah/Tacotron-2#322]
Hi there,

once again thanks for your amazing work. I've been trying multiple times to train a model with different parameters, and although I usually get decent results in general, the system is not robust in the sense that:

- For some sentences, specially for short ones (1 or 2 words), the model is not able to align well and the output is just noise. Sometimes, instead of noise, I get a very long pause at the beggining/end and then the spoken word.
- For some random sentences (not long nor short), the model is also not able to align well for random reasons and it outputs just noise.

I don't know if you guys have experienced the same inestability. My models are usually trained in a 3-2080Ti machine with batch size 30*3, and for about 100-150K steps before it starts overfitting.

Masking the encoder or not seems to affect the stability of the system, but in both cases I experience these kind of alignment problems while in synthesis mode.

Has anyone experienced this kind of problems?

**Comment by alexdemartos** [comment]
There might be something wrong with the padding/masking?
